Header header
int8 drone_id
int8 command